lib/infra/remote-store-resources.ts

```diff
@@ -11,7 +11,7 @@ export class RemoteStoreResources {
       this.snapshotS3Bucket = new Bucket(scope, `remote-store-${scope.stackName}`, {
         removalPolicy: RemovalPolicy.DESTROY,
         autoDeleteObjects: true,
-        bucketName: `${scope.stackName}`,
+        bucketName: `${scope.stackName.toLowerCase()}`,
```

Comment:
Hi @rishabh6788 ,

I saw this bug while adding test cases
```
    Bucket name must only contain lowercase characters and the symbols, period (.) and dash (-) (offset: 0)
    Bucket name must start and end with a lowercase character or number (offset: 0)
```
Hence changing this to lowercase. I wonder if we can remove the bucket name as each stack can track the associated bucket. Unless there is another use case that I might be unaware of